static/src/app/library/library.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+import leaflet from 'leaflet';
```

Comment:
The javascript for the library page isn't hooked up - All I did was paste in the html template.
I still need to set up the import for leaflet correctly - OTM has a good example in their config of how to bring libraries which don't respect webpack modules such as leaflet in as globals: it's like one line of code, but it's an extra step.  I'll add instructions on how to do so in the readme as well.